src/Components/Blueprints/ImportBlueprintModal.tsx

```diff
@@ -50,52 +53,63 @@ export const ImportBlueprintModal: React.FunctionComponent<
     _event: React.ChangeEvent<HTMLInputElement> | React.DragEvent<HTMLElement>,
     file: File
   ) => {
+    setFileContent('');
     setFilename(file.name);
     setIsRejected(false);
     setIsInvalidFormat(false);
   };
+  React.useEffect(() => {
+    if (filename && fileContent) {
+      try {
+        const isToml = filename.endsWith('.toml');
+        const isJson = filename.endsWith('.json');
+        if (isToml) {
+          const tomlBlueprint = parse(fileContent);
+          const blueprintFromFile = mapOnPremToHosted(tomlBlueprint);
+          const importBlueprintState = mapExportRequestToState(
+            blueprintFromFile,
+            []
+          );
+          setImportedBlueprint(importBlueprintState);
```

Comment:
Should we show some warning, that this is a best effort import?

Reply:
Let me know how you like that one I came up with. We can adjust it. :)